webhook_testing.go

```diff
@@ -0,0 +1,402 @@
+package braintree
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"time"
+
+	"text/template"
+)
+
+var payloadTemplate = `
+<notification>
+	<timestamp type="datetime">%s</timestamp>
+	<kind>%s</kind>
+	<subject>%s</subject>
+</notification>
+`
+
+// GetPayload returns string representation of XML payload, base64 encoded
+func (g *WebhookTestingGateway) GetPayload(kind, id string) string {
```

Comment:
Lets name this `SamplePayload` and call out in the comment that the function builds a sample payload of the given kind. It'll provide more clarity to what this function actually returns.